src/components/resizable_container/types.ts

```diff
@@ -16,6 +16,9 @@ export type PanelDirection = 'left' | 'right';
 
 export type KeyMoveDirection = 'forward' | 'backward';
 
+type ResizeTriggerTuple = ['pointer', 'key'];
+export type ResizeTrigger = ResizeTriggerTuple[number];
```

Comment:
Sorry, maybe I'm missing something, but if we're not using `ResizeTriggerTuple`, why not just do
```suggestion
export type ResizeTrigger = 'pointer' | 'key';
```

Reply:
No problem, it might make more sense with some screenshots. I found when exporting `ResizeTrigger` as a union directly, it would show the type in the docs, which seemed confusing to me since there's no way for a consumer to actually reference that internal type:
<img width="1097" alt="Screen Shot 2022-09-26 at 3 59 13 PM" src="https://user-images.githubusercontent.com/25592674/192358444-97686849-4ba0-49d7-b6e6-b21c1bd67a32.png">

I noticed other props used `keyof typeof` to pull the keys from an object which would show up in the docs, so I was trying to do a similar thing with `ResizeTrigger`:
<img width="1097" alt="Screen Shot 2022-09-26 at 3 59 59 PM" src="https://user-images.githubusercontent.com/25592674/192358489-8ad5c3d0-bdc2-41aa-bb18-6a44dd0dd5e3.png">

But this is a super minor thing, and I documented `'pointer' | 'key'` in the description anyway, so it might not matter to us at all. I'm totally fine with just using the union type directly instead if preferred.

Reply:
Ahhh gotcha! I always forget react docgen isn't always perfect in its output of our types 😅 To be honest, if it's already documented in the description, I'm not fussed about the type output, but I'm also totally cool to leave this as-is (maybe with an optional comment noting that it's written the way it is for the props table vs. dev readability). Either way, not a blocker, so I'll go ahead and approve this PR! 🎉

Reply:
I ended up switching it to a simpler type since the props table isn't an issue and it's more readable. Thanks for the feedback!